DanielVargas/tic_tac_toe/src/com/company/Tictactoe.java

```diff
@@ -0,0 +1,176 @@
+package com.company;
+import java.util.Scanner;
+
+public class Tictactoe {
```

Comment:
Please follow java convention to declare class names, when the name is compound for many words the first letter of each internal word needs to be capitalized as well. 

```java
public class JavaClassName {
}
```